src/DetailsView/reports/components/report-sections/details-section.tsx

```diff
@@ -12,6 +12,12 @@ export type DetailsSectionProps = Pick<SectionProps, 'pageUrl' | 'description' |
 
 export const DetailsSection = NamedSFC<DetailsSectionProps>('DetailsSection', props => {
     const { pageUrl, description, scanDate, toUtcString } = props;
+    const scanDateUTC: string = toUtcString(scanDate);
+    const screenReaderTexts = {
+        targetPageLink: `Target Page: ${pageUrl}`,
+        scanDate: `Scan date: ${scanDateUTC}`,
+        comment: `Comment: ${description}`,
+    };
     return (
         <div className="scan-details-section">
```

Comment:
Not sure why but by using JAWS and navigating to the table, the AT is reading the whole table content at once. Later, using arrow keys (down arrow key) you can navigate to the different cells to get JAWS to read the content per cell. Worth looking at it since our 2 internal AT users use JAWS not NVDA.

Reply:
As discussed, I am trying it by removing the spans from `tr` and then checking it with JAWS

Reply:
Fixed